tests/integration_lma/test_lma_bundle.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+import os
+import shutil
+import tempfile
+
+import pytest
+from git import Repo
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.abort_on_fail
+async def test_clone_lma_bundle_and_run_its_tests(ops_test):
+    charm_under_test = await ops_test.build_charm(".")
+    log.info("Built charm %s", charm_under_test)
+
+    with tempfile.TemporaryDirectory() as temp_charm_dir:
+        # Launching pytest from within pytest deletes the built *.charm because pytest-operator's
+        # `build_charm()` does `rmtree` if build path already exists
+        # https://github.com/charmed-kubernetes/pytest-operator/blob/main/pytest_operator/plugin.py
+        # Copying the built charm elsewhere to make sure it is still available when the lma bundle
+        # is being deployed.
+        shutil.copy(charm_under_test, temp_charm_dir)
+        charm_under_test = os.path.join(temp_charm_dir, os.path.split(charm_under_test)[-1])
+
+        with tempfile.TemporaryDirectory(dir=os.getcwd()) as temp_dir:
+            log.info("Cloning lma-light-bundle repo to a temp folder %s", temp_dir)
+            Repo.clone_from(
+                "https://github.com/canonical/lma-light-bundle.git",
```

Comment:
This needs to be parameterized. If I want to run this offline I need to be able to at least point to a local copy of the repo.

Reply:
Upon further reflection, `charmcraft build` also requires network. I would like to have offline tests working, but it won't be a blocker for me as it is a significant amount of work to do so.